functional_tests/identify.test.ts

```diff
@@ -33,109 +33,91 @@ test('identify sends a identify event', async () => {
 test('identify sends an engage request if identify called twice with the same distinct id and with $set/$set_once', async () => {
     // The intention here is to reduce the number of unncecessary $identify
     // requests to process.
-    const posthog = await createPosthogInstance()
+    const token = v4()
+    const posthog = await createPosthogInstance(token)
 
     const anonymousId = posthog.get_distinct_id()
 
     // The first time we identify, it calls the /e/ endpoint with an $identify
     posthog.identify('test-id', { email: 'first@email.com' }, { location: 'first' })
 
     await waitFor(() =>
-        expect(capturedRequests['/e/']).toContainEqual(
+        expect(getRequests(token)['/e/']).toContainEqual(
             expect.objectContaining({
                 event: '$identify',
                 $set: { email: 'first@email.com' },
                 $set_once: { location: 'first' },
                 properties: expect.objectContaining({
                     distinct_id: 'test-id',
                     $anon_distinct_id: anonymousId,
-                    token: posthog.config.token,
                 }),
             })
         )
     )
 
-    // The second time we identify, it calls the /engage/ endpoint with the $set
-    // / $set_once properties, in two separate requests.
+    // The second time we identify, it instead sents an event of type "$set".
     posthog.identify('test-id', { email: 'test@email.com' }, { location: 'second' })
 
     await waitFor(() =>
-        expect(capturedRequests['/engage/']).toContainEqual(
+        expect(getRequests(token)['/e/']).toContainEqual(
             expect.objectContaining({
-                $distinct_id: 'test-id',
-                $set: {
+                event: '$set',
+                properties: expect.objectContaining({
                     $browser: 'Safari',
                     $browser_version: null,
                     $referrer: '$direct',
                     $referring_domain: '$direct',
-                    email: 'test@email.com',
-                },
-                $token: posthog.config.token,
-            })
-        )
-    )
-
-    await waitFor(() =>
-        expect(capturedRequests['/engage/']).toContainEqual(
-            expect.objectContaining({
-                $distinct_id: 'test-id',
-                $set_once: { location: 'second' },
-                $token: posthog.config.token,
+                    $set: { email: 'test@email.com' },
```

Comment:
this seems a bit weird to me, shouldn't all these other properties go inside `$set` too? Otherwise these are set on the event, not on the person

Reply:
Yeah I'm not 100% on what is meant to happen here tbh

Reply:
yes, Neil is correct `$set` isn't special in any way... none of our events are ... they all just take properties from event.properties.$set